.github/buildomat/jobs/falcon-build.sh

```diff
@@ -18,6 +18,7 @@ rustc --version
 
 banner check
 ptime -m cargo check --features falcon
+ptime -m cargo xtask clippy
```

Comment:
Probably just scope this to a `cargo clippy ---features falcon`?  We don't need to run the non-falcon checks.

Reply:
done, also with `--all-targets` flag